### ISSUE TITLE
Usage in a Svelte REPL

### ISSUE THREAD

**Issue by jycouet** [BridgeAR/safe-stable-stringify#29]
I'm trying to use `stringify` in a Svelte REPL, but I get this message:
```
'stringify' is not exported by https://unpkg.com/safe-stable-stringify@2.3.1/index.js, imported by ./App.svelte
```
REPL: https://svelte.dev/repl/c1e15a7340294be3aa876091d8539709?version=3.46.4

I think that if you export this lib in a mjs format as well it will work?

**Comment by Properko** [comment]
Getting a similar error when `tsconfig.json` has `"moduleResolution": "Node16",`

```
> tsc

src/utils/logger.ts:3:27 - error TS7016: Could not find a declaration file for module 'safe-stable-stringify'. '{...}/package-creator-tool/node_modules/safe-stable-stringify/esm/wrapper.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/safe-stable-stringify` if it exists or add a new declaration (.d.ts) file containing `declare module 'safe-stable-stringify';`

3 import jsonStringify from "safe-stable-stringify";
                            ~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in src/utils/logger.ts:3
```

**Comment by pkerschbaum** [comment]
@Properko your problem is that the types of `safe-stable-stringify` are not properly configured for TS `node16` ESM mode. I created an issue for that #32 (and a pull request fixing it #33).

**Comment by BridgeAR** [comment]
@jycouet this should be fixed by v2.4.0. If you could verify that, that would be great!

**Comment by jycouet** [comment]
@BridgeAR I confirm, It's working.
Here is the new REPL link: https://svelte.dev/repl/30e5ed5ea0eb4545a16780c56e8284d7?version=3.50.1

FYI, while it was not there, I did my own! _(a bit naively, not working with deep circular...)_
And since I was here, I launched your compare to see, and I'm not "tooooo" bad ^^ (OK, missing features and slower... But was doing the job for me). 
👉 I'll simply redirect to this package now; well done 👍
